packages/cli/test/cases/eject.default/eject.default.spec.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Use Case
+ * Run Greenwood eject command to copy core configuration files
+ *
+ * User Result
+ * Should eject configuration files to working directory
+ *
+ * User Command
+ * greenwood eject
+ */
+const fs = require('fs-extra');
+const path = require('path');
+const expect = require('chai').expect;
+const TestBed = require('../../../../../test/test-bed');
+
+describe('Eject Greenwood With: ', function() {
+  let setup;
+
+  before(async function() {
+    setup = new TestBed();
+    this.context = await setup.setupTestBed(__dirname);
+  });
+
+  describe('Default Eject Option', function() {
+
+    before(async function() {
+      await setup.runGreenwoodCommand('eject');
+    });
+
+    it('should output webpack config files to project working directory', function() {
+      let configFiles = fs.readdirSync(__dirname);
+      
+      configFiles = configFiles.filter((file) => file !== 'node_modules' && file !== 'eject.default.spec.js');
+      
+      /* 
+      * 'webpack.config.common.js',
```

Comment:
here and in the next `describe`, can we please get an `it` per expected file?  (in addition to an `it` for the no. of files).